packages/constants/src/__test__/logLevels.test.js

```diff
@@ -0,0 +1,16 @@
+import { logLevels } from '..'
+
+describe('logLevels', () => {
+	describe('entries', () => {
+		it('should be camelCase', () => {
+			Object.keys(logLevels).forEach((logLevel) => {
+				expect(logLevel).toMatch(/^[a-z][a-zA-Z]*/)
+			})
+		})
+		it('should have a numeric value', () => {
+			Object.keys(logLevels).forEach((logLevel) => {
+				expect(logLevels[logLevel]).toEqual(logLevel)
```

Comment:
I don't understand the name of this test, that doesn't seem to be what it's testing. In fact what it seems to actually test is that the name of each log level exactly matches its value. (For instance, you can fail this test by adding `export const fart = 'fart123'` from `logLevels.js`, which I found out because I did.)

Can you double-check what you're trying to test here?

Reply:
I fixed the test but not the name!